frontend/include/chpl/types/Param.h

```diff
@@ -278,6 +281,19 @@ template<> struct stringify<chpl::types::Param::NoneValue> {
   }
 };
 
+template<> struct stringify<chpl::types::Param::EnumValue> {
+  void operator()(std::ostream& streamOut,
+                  chpl::StringifyKind stringKind,
+                  const chpl::types::Param::EnumValue& stringMe) const {
+    if (stringKind == chpl::StringifyKind::CHPL_SYNTAX) {
+      streamOut << stringMe.second;
+    } else {
+      streamOut << stringMe.second;
+      streamOut << " (" << stringMe.first.str() << ")";
+    }
+  }
+};
+
```

Comment:
This makes me a little nervous because it will apply to all pairs of IDs and strings (as opposed to just `EnumValue`, since `EnumValue` is a simple type alias. Consider making `EnumValue` its own struct. You can even make it wrap a `std::pair` if you want to make use of the existing implementation of `hash`, `mark`, etc defined on that type.

Reply:
> Another open question: should we use a struct with named fields instead of a pair, so that the code is more readable than using first and second?

Looks like there are other reasons to do this, so I think you should proceed with making a struct. Like I said, it could still wrap the pair, and provide `.first` and `.second` via getter methods.